typescript/src/lib/database/keys/settings/Channels/Logs.ts

```diff
@@ -5,8 +5,15 @@ export const EmojiCreate = 'channelsLogsEmojiCreate';
 export const EmojiDelete = 'channelsLogsEmojiDelete';
 export const EmojiUpdate = 'channelsLogsEmojiUpdate';
 export const Image = 'channelsLogsImage';
-export const Member = 'channelsLogsMember';
-export const Message = 'channelsLogsMessage';
+export const MemberAdd = 'channelsLogsMemberAdd';
+export const MemberNickNameUpdate = 'channelsLogsMemberNickNameUpdate';
+export const MemberRemove = 'channelsLogsMemberRemove';
+export const MemberRoleUpdate = 'channelsLogsMemberRoleUpdate';
```

Comment:
```suggestion
export const MemberRolesUpdate = 'channelsLogsMemberRolesUpdate';
```

Following earlier comments